colin-api/src/colin_api/models/business.py

```diff
@@ -671,3 +671,93 @@ def reset_corp_states(cls, cursor, event_ids: List):
         except Exception as err:
             current_app.logger.error(f'Error in Business: Failed reset ended corp_state rows for events {event_ids}')
             raise err
+
+    @classmethod
+    def find_by_corp_num(cls, corp_num: str, con=None) -> Business:
+        """Return a Business by corp_num."""
+        business = None
+        try:
+            # get record
+            if not con:
+                con = DB.connection
+                # con.begin()
+
+            cursor = con.cursor()
+            cursor.execute(
+                """
+                select corp.corp_num, corp_typ_cd, recognition_dts, bn_15, can_jur_typ_cd, othr_juris_desc,
+                    filing.period_end_dt, last_agm_date, corp_op_state.full_desc as state, admin_email,
+                    corp_state.state_typ_cd as corp_state, corp_op_state.op_state_typ_cd as corp_state_class
```

Comment:
we only need the following info returned based off of the requirements in your ticket.

```
{
  "business": {
    "identifier": "0111111",
    "legalName": "SCOTT LTD.",
    "legalType": "BC",
    "homeJurisdiction": null,
    "homeIdentifier": null
  }
}
```

Reply:
I think `homeJurisdiction` is `jurisdiction` but I am not sure what is `homeIdentifier`

Reply:
I need to check the database tables.  I think homeJurisdiction might be different than what you have.

Reply:
you are right.  we can just use `jurisdiction`.  I have included some separate updates req'd in different comments though.